docs/Rulebooks.md

```diff
@@ -203,10 +203,10 @@ You could write a rule that gives a role to `frenchDevs` like so (we assume that
 
 ```kotlin
 "FrenchDevsRule" % { email ->
-    val result = httpGetJson(
+    val result: Map<String, Any?> = httpGetJson(
         url = "https://myapi.com/api/endpoint?email=" + email.encodeURLQueryComponent(),
-        basicAuth = "apiusername" to "apipassword"
-    )
+        auth = Basic("apiusername", "apipassword") // Bearer(token) can also be used
+    ) // Return type can be whatever you want
```

Comment:
Please  make triple sure that the Rulebook documentation is accurate w.r.t. the latest changes

Reply:
It is